test/support/fixtures/xdr.ex

```diff
@@ -30,4 +31,14 @@ defmodule Stellar.Test.Fixtures.XDR do
 
   # set_trustline
   defdelegate set_trustline_flags(trustor, asset, clear_flags, set_flags), to: Trustline
+
+  # predicates
+  defdelegate predicate_unconditional(predicate), to: Predicate
```

Comment:
@einerzg `unconditional_predicate` would sound better for an English reader.

Reply:
I wrote this name because the type for this case is called `CLAIM_PREDICATE_UNCONDITIONAL`. Do you think it's better to change?

Reply:
What about `predicate_unconditional/1`?

Reply:
sounds good to me!!